Added Media Type (#213)
opengeospatial/geoparquet: format-specs/geoparquet.md

It is reasonable for implementations to require that one of the above `id` elements are present and skip further tests to determine if the CRS is functionally equivalent with OGC:CRS84.

Note: EPSG:4326 and OGC:CRS84 are equivalent with respect to this specification because this specification specifically overrides the coordinate axis order in the `crs` to be longitude-latitude.

## Media Type

> A GeoParquet file SHOULD use [`application/vnd.apache.parquet`](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as the [Media Type](https://en.wikipedia.org/wiki/Media_type) (previously known as the MIME Type).

Reviewer: ```suggestion
If a [media type](https://en.wikipedia.org/wiki/Media_type) (formerly: MIME type) is used, a GeoParquet file MUST use [application/vnd.apache.parquet](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as media type.
```

> If a [media type](https://en.wikipedia.org/wiki/Media_type) (formerly: MIME type) is used, a GeoParquet file MUST use [application/vnd.apache.parquet](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as media type.

Author: ```suggestion
If a [media type](https://en.wikipedia.org/wiki/Media_type) (formerly: MIME type) is used, a GeoParquet file MUST use [application/vnd.apache.parquet](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as the media type.
```

> A GeoParquet file ([RFC 8259](https://tools.ietf.org/html/rfc8259)) SHOULD use [`application/vnd.apache.parquet`](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as the [Media Type](https://en.wikipedia.org/wiki/Media_type) (previously known as the MIME Type).

Reviewer: Why don't we go for a MUST here?

The RFC link seems to be out of place?

Reviewer: @cholmes You edited my post 😂 Happy that it's not just me who does this by accident ;-)

Author: Double fail! I'll just leave this be then. New proposal looks good, I'll add it.

> A GeoParquet file ([RFC 8259](https://tools.ietf.org/html/rfc8259)) SHOULD use [`application/vnd.apache.parquet`](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as the [Media Type](https://en.wikipedia.org/wiki/Media_type) (previously known as the MIME Type).

Author: ```suggestion
A GeoParquet file SHOULD use [`application/vnd.apache.parquet`](https://www.iana.org/assignments/media-types/application/vnd.apache.parquet) as the [Media Type](https://en.wikipedia.org/wiki/Media_type) (previously known as the MIME Type).
```
